packages/rtk-query-codegen-openapi/package.json

```
@@ -58,7 +58,7 @@
     "@apidevtools/swagger-parser": "^10.0.2",
     "@rtk-query/oazapfts-patched": "^3.6.0-2",
     "commander": "^6.2.0",
-    "prettier": "^2.2.1",
+    "prettier": "^2.8.8",
```

Is there any technical reason to bump this? It will require all of our consumers to update their prettier installation. If anything, I'd suggest to change this
```suggestion
    "prettier": "^2.2.1 | ^3.0.0",
```
(but we'd need to test that 3.0 actually is working with the api we use)

---

The 3.0.0 version is esm, and the scope of modification is much larger.
If we use the new syntax, the current prettier cannot handle it. Is there any reason to keep it fixed at a lower version?

---

Ah, then 3.0 is a bit in the future I guess :)
From a package maintainer perspective: there is just no good reason to bump a dependency ever - `^2.2.1` still allows consumers to use `2.8.8` in their codebase - but bumping it up interrupts our users. So I'd say let's keep the dependencies at their current version.

---

okay

---

done